lib/instance.js

```diff
@@ -147,6 +147,65 @@ function instance(system) {
 		}
 	})
 
+	self.system.on('instance_http_req', function (req, res) {
+		let instance
+		res.set('Content-Type', 'text/plain')
```

Comment:
Are modules able to override this if it is done here?

Reply:
Yes, modules can specify whatever headers they like, and if they specify a 'Content-Type' of something else, such as `application/json` then that'll overwrite the `text/plain`.